File: 02_ProgrammingFundamentals/_03_Arrays/_02_Exercise/_02_CommonElements.java
package _02_ProgrammingFundamentals._03_Arrays._02_Exercise;

import java.util.Scanner;

public class _02_CommonElements {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input values
        String[] arrOne = scan.nextLine().split("\\s+");
        String[] arrTwo = scan.nextLine().split("\\s+");

        // Print common elements in two arrays
        for (int i = 0; i < arrOne.length; i++) {
            for (int j = 0; j < arrTwo.length; j++) {

                if (arrOne[i].equals(arrTwo[j])) {
                    System.out.print(arrTwo[j] + " ");
                }
            }
        }

        scan.close();
    }
}

File: 24_C#Fundamentals/_03_Arrays/_02_Exercise/_02_CommonElements/_02_CommonElements.cs
using System;

public class _02_CommonElements
{
    public static void Main(string[] args)
    {
        // Read input values
        string[] arrOne = Console.ReadLine().Split(new char[] { ' ' }, StringSplitOptions.RemoveEmptyEntries);
        string[] arrTwo = Console.ReadLine().Split(new char[] { ' ' }, StringSplitOptions.RemoveEmptyEntries);

        // Print common elements in two arrays
        for (int i = 0; i < arrOne.Length; i++)
        {
            for (int j = 0; j < arrTwo.Length; j++)
            {

                if (arrOne[i].Equals(arrTwo[j]))
                {
                   Console.Write(arrTwo[j] + " ");
                }
            }
        }
    }
}
